src/locales/en/translation.json

```diff
@@ -242,7 +242,8 @@
   "migrateV0Extension": {
     "backupInvalidPrivateKeys": {
       "confirm": "I’ve safely stored my private keys",
-      "description": "Some of your private keys have typos and won’t be stored by the new wallet extension. Please make sure to copy them and store them elsewhere before proceeding.",
+      "description": "{{count}} of your private keys have typos and won’t be stored by the new wallet extension. Please make sure to copy them and store them elsewhere before proceeding, so you can correct them later.",
```

Comment:
here we want singular form (keys-key, them-it etc), right?

Reply:
we can add that too :shrug: 1 sec